### DISCUSSION TITLE
Task list progress reports

### DISCUSSION THREAD

**Discussion by jwindmuller** [community/community#4248] [Other Feature Feedback, Questions, & Ideas]
We use tasks in issues to track the work that needs to get done and it's also been a good indicator of how fast the team moves. Right now we take a snapshot of task counts every morning and keep track of evolution that way.

This idea is about having this kind of report accessible natively at the milestone level and project. The main points we look at are:

1. How many tasks are on this milestone/project
2. How many tasks were added since yesterday (and in which issues)
3. How many tasks where checked off since yesterday (and in which issues)

Having the information per issue would be amazing:

1. Knowing which issues grew their number of tasks is very useful to analise what and why we missed something
2. Knowing which issues have slowed down let's us make sure that the people working on it aren't stuck

And as an extra credit, if we could have the same report from a issue search that would be super powerful as well for project planning.

**Comment by tallguyjenks** [comment]
ooo a kind of burndown report or other reporting methods of these items would be very value added for teams using :octocat: kinda like Azure DevOps dashboards based on workitem completions
